kratos/python/add_node_to_python.cpp

```diff
@@ -141,6 +141,16 @@ void  AddNodeToPython(pybind11::module& m)
     ;
 
     py::class_<Dof<double>>(m,"Dof")
+    .def("GetVariable", &Dof<double>::GetVariable, py::return_value_policy::reference_internal)
```

Comment:
Finally!!!, we should expose the construction of MPC to python now that dof are fully available in python. Will be very practical

Reply:
ok, but only after this PR is merged